IdokladSdk/Validation/Attributes/NotEmptyStringAttribute.cs

```diff
@@ -1,12 +1,13 @@
 ﻿using System.ComponentModel.DataAnnotations;
+using System.Text.RegularExpressions;
 
 namespace IdokladSdk.Validation.Attributes
 {
     public class NotEmptyStringAttribute : ValidationAttribute
```

Comment:
Když už jsme to sjednocovali s API, mohlo se sjednotit i jméno toho atributu - v API to je NotEmptyOrWhiteSpace.